README.md

```diff
@@ -97,6 +97,14 @@ Options:
                            First extension will always be preferred to homonyms
                            with another allowed extension.
                                                        [array] [default: ["js"]]
+  --implicitDefault, -m    Uses defaults as implicit instead of named.
```

Comment:
Please use dashes for CLI program parameters, i.e. `--implicit-default`.

Reply:
@gajus - Are you sure? I was following the convention outlined in ignoreDirectories.

Reply:
Right. That should not have been the case originally. Please keep the convention. I will release a breaking change later.

Reply:
@gajus - Sounds good. If everything else is okay, I'll make some tests over the next couple days to check for any funny business between the various flags.

Reply:
What is the status of this PR?  I would love this change.